addons/event_sale/models/sale_order.py

```diff
@@ -174,12 +170,14 @@ def _get_sale_order_line_multiline_description_sale(self):
 
     def _get_display_price(self):
         if self.event_ticket_id and self.event_id:
-            # FIXME this is strange
-            # price_reduce is the price after discount
-            # shouldn't we leave the discount computation to sale
-            # and use the non reduced price here (aka price field)
-            return self.event_ticket_id.with_context(
+            event_ticket = self.event_ticket_id.with_context(
                 pricelist=self.order_id.pricelist_id.id,
-                uom=self.product_uom.id).price_reduce
-        else:
-            return super()._get_display_price()
+                uom=self.product_uom.id
+            )
+            company = event_ticket.company_id or self.env.company
+            currency = company.currency_id
+            return currency._convert(
+                event_ticket.price_reduce, self.order_id.currency_id,
+                self.order_id.company_id or self.env.company.id,
+                self.order_id.date_order or fields.Date.today())
```

Comment:
Shouldn't pricelist with discount have a different behavior than the other ones when getting the displayed price? :thinking:

Reply:
@Feyensv told me it's better not to change the way discounts are dealt with, that's the reason I didn't backport this part.